bin/compile

```diff
@@ -41,12 +41,12 @@ dotnet --info
 PROJECT_FILE=$(x=$(dirname $(find ${BUILD_DIR} -maxdepth 5 -iname Startup.cs | head -1)); while [[ "$x" =~ $BUILD_DIR ]] ; do find "$x" -maxdepth 1 -name *.csproj; x=`dirname "$x"`; done)
 PROJECT_NAME=$(basename ${PROJECT_FILE%.*})
 
-echo "restore ${PROJECT_FILE}"
-dotnet restore $PROJECT_FILE --runtime linux-x64
```

Comment:
Removed dotnet restore here because dotnet restore is now an implicit command, dotnet publish internally calls dotnet restore, no needs explicit call anymore.

Reference: https://github.com/dotnet/announcements/issues/23